### PR TITLE
Add legal licenses and copyrights

### PR DESCRIPTION
**Pull request by ELENAGER**
Signed-off-by: Elena Gershkovich <elenage@il.ibm.com>

### CODE REVIEW

**Review by matancarmeli7 — COMMENTED** [review]
Looks ok but I think we should add [this](https://github.com/IBM/ibm-block-csi-operator/blob/develop/hack/update-copyright.sh) script as part of this ticket and then we won't have to add automatically the copyrights, but if you don't have time due to the GA we can postpone it to do it after it

File: hack/update-copyright.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
```
Comment (matancarmeli7) [inline hack/update-copyright.sh:1]: can you open a ticket that will be part of our `make update` command?
and open another ticket to add to our CI that we will add a [check generated files](https://github.com/IBM/ibm-block-csi-operator/blob/develop/Makefile#L79-L80) step that will verify for each branch that we won't miss copyrights??

File: hack/update-copyright.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Add the copyright header for all fo files if not.
+
+CURRENT_PATH=$(dirname "$BASH_SOURCE")
+PROTECT_ROOT=$CURRENT_PATH/..
+VENDOR_PATH=$CURRENT_PATH/../vendor
+BOILERPLATE=$CURRENT_PATH/boilerplate.go.txt
```
[the remark this answers is not in the document]
Comment (ELENAGER) [reply hack/update-copyright.sh:24]: Done

File: hack/update-copyright.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  else
+    if !(contains $file_name "${excluded_files[@]}")
+    then
+      cat $BOILERPLATE > $file.tmp;
+      echo "" >> $file.tmp;
+      cat $file >> $file.tmp;
+      mv $file.tmp $file;
+    fi
+  fi
+done
```
[the remark this answers is not in the document]
Comment (ELENAGER) [reply hack/update-copyright.sh]: Done

File: hack/update-copyright.sh [hunk 4]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+
+#
+# Copyright 2019 IBM Corp.
```
[the remark this answers is not in the document]
Comment (ELENAGER) [reply hack/update-copyright.sh]: Done

File: hack/update-copyright.sh (hunk 1 above)
Comment (ELENAGER) [reply hack/update-copyright.sh:1]: Opened CSI-5233 and CSI-5234

Comment (matancarmeli7) [reply hack/update-copyright.sh:1]: thanks

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**